ICSharpCode.CodeConverter/CSharp/MethodBodyExecutableStatementVisitor.cs

```diff
@@ -587,7 +588,18 @@ public override async Task<SyntaxList<StatementSyntax>> VisitSelectBlock(VBSynta
                     csBlockStatements.Add(SyntaxFactory.BreakStatement());
                 }
                 var list = SingleStatement(SyntaxFactory.Block(csBlockStatements));
-                sections.Add(SyntaxFactory.SwitchSection(SyntaxFactory.List(labels), list));
+                var labelDesc = string.Join(" ", labels.Select(l => l.ToFullString()));
+                var numMatch = cases.Count(c => c == labelDesc);
+                if (numMatch == 0) {
+                    cases.Add(labelDesc);
+                    sections.Add(SyntaxFactory.SwitchSection(SyntaxFactory.List(labels), list));
+                } else {
+                    //Else write comment to code?
+                    var lastCase = sections.Last();
```

Comment:
If there were 3 duplicates, we'd actually want to comment out all except the first